act/qc/clean.py

```diff
@@ -749,6 +756,28 @@ def clean_arm_qc(
                             qc_var_name=qc_var_name, test_number=test_to_remove
                         )
 
+        # If the QC was not cleaned up because it is not correctly formatted with SERI QC
+        # call the SERI QC method.
+        try:
+            text = 'DQMS is composed of a suite of tests which includes SERI QC'
+            SERI_QC = text in self._ds.attrs['comment']
+        except KeyError:
+            SERI_QC = False
+
+        if SERI_QC and global_qc is None and qc_attributes is None:
+            self._ds.clean.clean_seri_qc()
+
+        # If the QC was not cleaned up because it is not correctcly formatted with
```

Comment:
correctcly to correctly

Reply:
@zssherman  let's hold off on merging. I'm doing some additional testing. Should I cancel this PR and submit a new one soon when it is ready to merge? Is there a way to temporarily stop it from running tests?

Reply:
Sounds good! And the tests should be stopped now.